### PR TITLE
Squish allows 

### PR DESCRIPTION
**Pull request by ayatnkw**
## Changes 
- Rephrasing sentences using 'allows' and anthropomorphism. 

## Related tickets and issues
[Doc-631](https://linear.app/n8n/issue/DOC-631/squish-the-allows)

### CODE REVIEW

**Comment by ayatnkw** [comment]
I'm not sure if it's easier to review in smaller chunks or do it all in one go 🤔 @StarfallProjects

File: _snippets/integrations/builtin/node-operations.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ When you add a node to a workflow, n8n displays a list of available operations.
 There are two types of operation:
 
 * Triggers start a workflow in response to specific events or conditions in your services. When you select a Trigger, n8n adds a trigger node to your workflow, with the Trigger operation you chose pre-selected. When you search for a node in n8n, Trigger operations have a bolt icon <span class="inline-image">![Trigger icon](/_images/common-icons/trigger.png){.off-glb}</span>.
-* Actions are operations that represent specific tasks or actions within a workflow, allowing you to manipulate data, perform operations on external systems, and trigger events in other systems as part of your workflows. When you select an Action, n8n adds a node to your workflow, with the Action operation you chose pre-selected.
+* Actions are operations that represent specific tasks or actions within a workflow which you can use to manipulate data, perform operations on external systems, and trigger events in other systems as part of your workflows. When you select an Action, n8n adds a node to your workflow, with the Action operation you chose pre-selected.
```
Comment (StarfallProjects) [inline _snippets/integrations/builtin/node-operations.md:8]:
```suggestion
* Actions are operations that represent specific tasks or actions within a workflow that you can use to manipulate data, perform operations on external systems, and trigger events in other systems as part of your workflows. When you select an Action, n8n adds a node to your workflow, with the Action operation you chose pre-selected.
```

Comment (StarfallProjects) [reply _snippets/integrations/builtin/node-operations.md:8]: 'which' has a comma before it

https://www.dictionary.com/e/that-vs-which/

File: docs/advanced-ai/examples/api-workflow-tool.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ This workflow uses:
 
 * [Chat Trigger](/integrations/builtin/core-nodes/n8n-nodes-langchain.chattrigger/): start your workflow and respond to user chat interactions. The node provides a customizable chat interface.
 * [Agent](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.agent/): the key piece of the AI workflow. The Agent interacts with other components of the workflow and makes decisions about what tools to use.
-* [Custom n8n Workflow Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/): plug in n8n workflows as custom tools. In AI, a tool is an interface the AI can use to interact with the world (in this case, the data provided by your workflow). It allows the AI model to access information beyond its built-in dataset.
+* [Custom n8n Workflow Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/): plug in n8n workflows as custom tools. In AI, a tool is an interface the AI can use to interact with the world (in this case, the data provided by your workflow). Use the AI model to access information beyond its built-in dataset.
```
Comment (StarfallProjects) [inline docs/advanced-ai/examples/api-workflow-tool.md:23]:
```suggestion
* [Custom n8n Workflow Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/): plug in n8n workflows as custom tools. In AI, a tool is an interface the AI can use to interact with the world (in this case, the data provided by your workflow). The AI model uses the tool to access information beyond its built-in dataset.
```

File: docs/advanced-ai/examples/data-google-sheets.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ This workflow uses:
 
 * [Chat Trigger](/integrations/builtin/core-nodes/n8n-nodes-langchain.chattrigger/): start your workflow and respond to user chat interactions. The node provides a customizable chat interface.
 * [Agent](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.agent/): the key piece of the AI workflow. The Agent interacts with other components of the workflow and makes decisions about what tools to use.
-* [Custom n8n Workflow Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/): plug in n8n workflows as custom tools. In AI, a tool is an interface the AI can use to interact with the world (in this case, the data provided by your workflow). It allows the AI model to access information beyond its built-in dataset.
+* [Custom n8n Workflow Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/): plug in n8n workflows as custom tools. In AI, a tool is an interface the AI can use to interact with the world (in this case, the data provided by your workflow). Use the AI model to access information beyond its built-in dataset.
```
Comment (StarfallProjects) [inline docs/advanced-ai/examples/data-google-sheets.md:23]:
```suggestion
* [Custom n8n Workflow Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/): plug in n8n workflows as custom tools. In AI, a tool is an interface the AI can use to interact with the world (in this case, the data provided by your workflow). The AI model uses the tool to access information beyond its built-in dataset.
```

File: docs/code/builtin/current-node-input.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ You can use Python in the Code node. It isn't available in expressions.
 	| `$input.last()` | Last input item in current node. | :white_check_mark: |
 	| `$input.params` | Object containing the query settings of the previous node. This includes data such as the operation it ran, result limits, and so on.  | :white_check_mark: |
 	| `$json` | Shorthand for `$input.item.json`. Incoming JSON data from a node. Refer to [Data structure](/data/data-structure/) for information on item structure. | :white_check_mark: (when running once for each item) |
-	| `$input.context.noItemsLeft` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node, allowing you to see if the node is still processing items. | :white_check_mark: |
+	| `$input.context.noItemsLeft` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node, helping you determine whether the node is still processing items. | :white_check_mark: |
```
Comment (StarfallProjects) [inline docs/code/builtin/current-node-input.md:25]:
```suggestion
	| `$input.context.noItemsLeft` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node. Use this to determine whether the node is still processing items. | :white_check_mark: |
```

File: docs/code/builtin/current-node-input.md [hunk 5]
```diff
@@ -32,4 +32,4 @@ You can use Python in the Code node. It isn't available in expressions.
 	| `_input.last()` | Last input item in current node. | 
 	| `_input.params` | Object containing the query settings of the previous node. This includes data such as the operation it ran, result limits, and so on.  | 
 	| `_json` | Shorthand for `_input.item.json`. Incoming JSON data from a node. Refer to [Data structure](/data/data-structure/) for information on item structure. Available when you set **Mode** to **Run Once for Each Item**. | 
-	| `_input.context.noItemsLeft` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node, allowing you to see if the node is still processing items. | 
+	| `_input.context.noItemsLeft` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node, helping you determine whether the node is still processing items. |
```
Comment (StarfallProjects) [inline docs/code/builtin/current-node-input.md:35]:
```suggestion
	| `_input.context.noItemsLeft` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node. Use this to determine whether the node is still processing items. | 
```

File: docs/code/builtin/output-other-nodes.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ You can use Python in the Code node. It isn't available in expressions.
 	| `$("<node-name>").last(branchIndex?, run Index?)` | The last item output by the given node. | :white_check_mark: |
 	| `$("<node-name>").item` | The linked item. This is the item in the specified node used to produce the current item. Refer to [Item linking](/data/data-mapping/data-item-linking/) for more information on item linking. | :x: |
 	| `$("<node-name>").params` | Object containing the query settings of the given node. This includes data such as the operation it ran, result limits, and so on. | :white_check_mark: |
-	| `$("<node-name>").context` | Only available when working with the Loop Over Items node. Provides information about what's happening in the node, allowing you to see if the node is still processing items. | :white_check_mark: |
+	| `$("<node-name>").context` | Only available when working with the Loop Over Items node. Provides information about what's happening in the node, helping you determine whether the node is still processing items. | :white_check_mark: |
```
Comment (StarfallProjects) [inline docs/code/builtin/output-other-nodes.md:23]:
```suggestion
	| `$("<node-name>").context` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node. Use this to determine whether the node is still processing items. | :white_check_mark: |
```

File: docs/code/builtin/output-other-nodes.md [hunk 7]
```diff
@@ -30,5 +30,5 @@ You can use Python in the Code node. It isn't available in expressions.
 	| `_("<node-name>").last(branchIndex?, run Index?)` | The last item output by the given node. | :white_check_mark: |
 	| `_("<node-name>").item` | The linked item. This is the item in the specified node used to produce the current item. Refer to [Item linking](/data/data-mapping/data-item-linking/) for more information on item linking. | :x: |
 	| `_("<node-name>").params` | Object containing the query settings of the given node. This includes data such as the operation it ran, result limits, and so on. | :white_check_mark: |
-	| `_("<node-name>").context` | Only available when working with the Loop Over Items node. Provides information about what's happening in the node, allowing you to see if the node is still processing items. | :white_check_mark: |
+	| `_("<node-name>").context` | Only available when working with the Loop Over Items node. Provides information about what's happening in the node, helping you determine whether the node is still processing items. | :white_check_mark: |
```
Comment (StarfallProjects) [inline docs/code/builtin/output-other-nodes.md:33]:
```suggestion
	| `_("<node-name>").context` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node. Use this to determine whether the node is still processing items. | :white_check_mark: |
```

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
See inline comments

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**